core/src/OpenMP/Kokkos_OpenMP_Team.hpp

```diff
@@ -169,7 +177,12 @@ class TeamPolicyInternal<Kokkos::OpenMP, Properties...>
  public:
   inline int team_size() const { return m_team_size; }
   inline int league_size() const { return m_league_size; }
-
+  inline bool auto_team_size() const { return m_tune_team; }
+  inline bool auto_vector_length() const { return m_tune_vector; }
```

Comment:
Same as above: if there is not a good reason to expose this make them impl_

Reply:
done